src/Config/CS_SDK.props

```diff
@@ -49,21 +49,21 @@
     <GenerateDependencyFile>false</GenerateDependencyFile>
   </PropertyGroup>
   <PropertyGroup Condition="$(Platform.Contains('AnyCPU'))" >
-    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
```

Comment:
should we support older versions of windows ? (win7 ?)

Reply:
Windows 7 and Windows 8.1 are currently [supported with .NET 6](https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md#windows). They will not be supported with .NET 7+.
Ok...I guess that is decided

Reply:
I will probably have to go back on this decision when adding net8